assets/lambda/new_s3obj_queue_notifier_lambda.py

```diff
@@ -12,8 +12,7 @@
 
 sqs_client = boto3.client("sqs")
 
-# TODO: need to be able to parameterize the region for our whole cape setup...
-ddb_resource = boto3.resource("dynamodb", region_name="us-east-2")
+ddb_resource = boto3.resource("dynamodb", region_name=os.getenv("AWS_REGION"))
```

Comment:
Turns out that the `AWS_REGION` variable already exists and is specific to the region in which the lambda runs. This is not actually what we want, so we need to change the name of this variable. I suggest `DDB_REGION`

This applies to *all* uses of `AWS_REGION` in this PR.